tests/main/test_homePage.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+
+@pytest.mark.django_db
+class TestHomepageView:
+    def test_user_redirected_to_admin_login(self, client):
+        response = client.get('/admin')
+        assert response.status_code == 301
```

Comment:
Maybe we can also check the response URL?

Reply:
Done